Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -252,8 +253,6 @@ public override void OnPreSceneQuery()
 
                     Vector3 endPoint = newGazeOrigin + (newGazeNormal * pointerExtent);
                     Rays[0].UpdateRayStep(ref newGazeOrigin, ref endPoint);
-
-                    gazeProvider.HitPosition = Rays[0].Origin + (gazeProvider.lastHitDistance * Rays[0].Direction);
```

Comment:
What change in behavior does this lead to (if any)?

Reply:
Since this value is also set in https://github.com/microsoft/MixedRealityToolkit-Unity/blob/2c363bd001736fd58e5c4a7de33464a6269a6103/Assets/MRTK/Services/InputSystem/GazeProvider.cs#L549-L573, removing this line didn't observably change any behavior. What it does is better keep the gaze pointer separate from the gazeProvider implementation, making the relationship between the two more straight forward (though there are still some lingering dependencies)